### ISSUE TITLE
Allow curl command display without "try" button

### ISSUE THREAD

**Issue by guideloom** [rapi-doc/RapiDoc#904]
Would be great to have the "curl" example showing, without the try button. I had assumed by using the following combination of options that this would be achievable:

`allow-try : false`
`show-curl-before-try: true`

However, it seems the curl display is completely dependent of "allow-try" being set to true, which also shows the "try" button.

**Comment by mrin9** [comment]
thats true `show-curl-before-try` depends on `allow-try` thats why both attributes have the word `try` in it . The reason is the `curl` syntax is dynamically generated based on what the user is typing in the input boxes before firing the try command.

Now before anything typed into those input boxes the `curl` will contain an  incomplete request. And in case you disabled `TRY` then there will be no input boxes and the `curl` will always be incorrect or incomplete.

I will still keep it open to see if we can generate a complete `curl` syntax from the schema info and the examples provided but it will be completely different  syntax compared to `show-curl-before-try` which mimics the actual user input
